src/main/java/io/split/android/client/events/SplitEvent.java

```diff
@@ -7,5 +7,6 @@
 public enum SplitEvent {
     SDK_READY,
     SDK_READY_FROM_CACHE,
-    SDK_READY_TIMED_OUT
+    SDK_READY_TIMED_OUT,
+    SDK_UPDATED
```

Comment:
Javi, check the names of these events in JS: https://help.split.io/hc/en-us/articles/360020448791-JavaScript-SDK#advanced-subscribe-to-events-and-changes
SDK_UPDATED --> SDK_UPDATE

Reply:
Fixed!